flink-table/flink-table-common/src/test/java/org/apache/flink/table/api/TableSchemaTest.java

```diff
@@ -53,6 +54,7 @@ public void testTableSchema() {
 				DataTypes.FIELD("q1", DataTypes.STRING()),
 				DataTypes.FIELD("q2", DataTypes.TIMESTAMP(3))))
 			.field("f2", DataTypes.STRING())
+			.field("f3", DataTypes.BIGINT(), "f0 + 1")
```

Comment:
Please add assertions for this computed column, e.g. assert the expected type and name for "f3" or 3th field.